### ISSUE TITLE
Site Migration: JITM for users who try to import a playground/full-site zip file

### ISSUE THREAD

**Issue by markbiek** [Automattic/wp-calypso#93219]
When a user tries to import a .zip file that contains plugins/themes/etc (eg from a WP playground or other full-site backup), we take them to the step to buy a Creator plan.

There is no warning which could be confusing/frustrating since we presented imports as being free. We should display a message when they upload a playground .zip.

This issue is to create/discuss a design for the new messaging and then implement that design.

P2 discussion: pfuQfP-yp-p2#comment-1032
Slack discussion: p1722606483011049-slack-C0Q664T29

**Comment by markbiek** [comment]
cc @fditrapani

**Comment by fditrapani** [comment]
Thanks @markbiek. I'll try work on a design next week!

**Comment by fditrapani** [comment]
I took some time to evaluate some options for this and here's how I think we should handle it:

After someone imports a playground or backup file, we show them a warning on the import screen that they need to upgrade: 

![image](https://github.com/user-attachments/assets/ecb96a2c-fe31-40e6-a03a-9d4847325675)

Pressing upgrade would take them to checkout and then return them back to this screen to complete the import. 

Upon further inspection, I noticed we can't actually import Playground files — even with a Creator/Business plan. Something seems to be broken there. If the scope of fixing that is too big a lift, an alternative approach could be to tell people to migrate their site instead:

![image](https://github.com/user-attachments/assets/f9315610-cd28-4428-877b-06954002ebc1)

From there, we ask for the upgrade and provide instructions for how to migrate.

**Comment by markbiek** [comment]
I think the second option where we say "Playground files are not supported by our importer" is the way to go. I don't think the importer is designed to handle themes/plugins at all (which is why we have migrations instead).

**Comment by fditrapani** [comment]
Awesome! Let's go with that then. Let me know if you need anything from my end.

**Comment by markbiek** [comment]
> Awesome! Let's go with that then. Let me know if you need anything from my end.

Will do! Thanks for jumping on that UI change!

**Comment by fditrapani** [comment]
We'll probably want to sync up some updates to our support docs because it's currently telling people they can import their Playground files which doesn't seem to be the case. 

https://wordpress.com/support/import/

We should probably also remove the mention of the Playground zip under the import field. What do you think>

**Comment by bohdankit07** [comment]
Let's look at the usage data first. If the numbers are low, it may be easier to turn it off completely until we find a fix and higher priorities are taken care of.

**Comment by zaerl** [comment]
>  I don't think the importer is designed to handle themes/plugins at al
Yes it does install everything possible. But the importers seems blocked now. A lot of changes have been made with migrations and co. so probably there is a bug somewhere.

**Comment by andres-blanco** [comment]
@markbiek I've assigned you to this issue since you are the last from the team that worked on this. What's the status? should this still be open?

**Comment by markbiek** [comment]
![Image](https://github.com/user-attachments/assets/5dd06008-8468-420c-bd75-1363f3570dbd)

I ran a quick funnel just to see how many people were coming into the hosted-site-migration flow from the above link and I didn't see any activity in the last 60 days.

Seems like turning it off completely is probably the way to go but I'm not sure what that would entail. Would it just be removing the stuff in the above screenshot from that support page?
